HomeTask6/app/src/app/components/login/login.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl,FormGroup } from '@angular/forms';
+import {Router} from '@angular/router';
+import { NewsService } from 'src/app/services/news.service';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+
+  username : string;
+  invalidUser: string;
+  loginForm = new FormGroup({
+    username : new FormControl(''),
+    password : new FormControl('')
+    });
+
+  constructor(private router: Router, private newsService: NewsService) { 
+    
+  }
+
+  ngOnInit() { }
+
+  loginCheck()
+  {
+    if(this.loginForm.get('username').value != '' && this.loginForm.get('password').value != '')
```

Comment:
user formcontrol required and in submit use *ngIf with formcontrol invalid keyword to skip this.

Reply:
Done